Task 10(3)/Task 10(3)/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace _3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double eps;
+            Console.Write("Введите eps: ");
+            eps = Convert.ToDouble(Console.ReadLine());
+            double aPrev = 1; 
+            double a = 2; 
+			double chislA = 1, chislB = 2, znamA = 1, znamB = 1; 
+            while (Math.Abs(a - aPrev) <= eps) 
```

Comment:
В задании сказано
 
> Написать программу, которая определяет первый член последовательности, который **отличается** от предыдущего по абсолютной величине **не более чем** на епсилон.

То есть, вычисляем члены последовательности пока разность больше. Как только станет не больше, прекращаем.
```suggestion
            while (Math.Abs(a - aPrev) > eps) 
```